### DISCUSSION TITLE
Radix is not in GitHub's search results

### DISCUSSION THREAD

**Discussion by scalisi** [radix-ui/primitives#1775] [Ideas] [answered]
The radix-ui/primitives repo is not in GitHub's search results when searching for "Radix":

https://github.com/search?q=Radix

Consider adding "Radix" to the About text (e.g., "Radix Primitives is an open-source UI component library...").

**Comment by andy-hook** [comment, marked as answer]
Thanks raising this @scalisi , I've updated our description!
